create_quiz/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,8 @@
+<resources>
+    <string name="app_name">Create_quiz</string>
+    <string name="nextButtonTxt">Next</string>
+    <string name="quizTitleTxt">QUIZ</string>
```

Comment:
We can use `android:textAllCaps="true"` when using the string, so "QUIZ" can be written as "Quiz" in case we change the design later.